src/sentry/utils/integrationdocs.py

```diff
@@ -122,3 +122,9 @@ def sync_integration_docs(platform_id, integration_id, path, quiet=False):
     key = get_integration_id(platform_id, integration_id)
 
     dump_doc(key, {"id": key, "name": data["name"], "html": data["body"], "link": data["doc_link"]})
+
+
+def integration_doc_exists(integration_id):
```

Comment:
I'd just add a comment similar to what I did suggesting why we need to do it this way, in case it comes up and someone asks, since it's definitely more inefficient, but safer.